package.json

```diff
@@ -1,164 +1,18 @@
 {
-  "name": "@livechat/design-system",
-  "version": "0.5.6",
-  "publishConfig": {
-    "access": "public"
-  },
-  "description": "LiveChat Design System React components with css styles",
-  "homepage": "https://livechat.github.io/design-system/",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/livechat/design-system"
-  },
-  "typings": "src/interfaces/index.d.ts",
-  "main": "dist/design-system.cjs.js",
-  "module": "dist/design-system.es.js",
-  "jsnext:main": "dist/design-system.es.js",
-  "style": "dist/design-system.css",
-  "files": [
-    "dist",
-    "src"
-  ],
+  "name": "design-system",
+  "private": true,
   "scripts": {
-    "start": "rollup -c --watch",
-    "build": "rollup -c",
-    "prebuild": "rimraf dist",
-    "build:cdn": "npm run build && mkdir -p dist/cdn && cp dist/design-system.css dist/cdn/styles.css && cp dist/cdn/styles.css dist/cdn/styles-v$npm_package_version.css",
-    "prepare": "npm run build",
-    "styleguide": "cross-env NODE_ENV=development styleguidist server",
-    "styleguide:build": "cross-env NODE_ENV=production styleguidist build",
-    "predeploy": "npm run-script styleguide:build",
-    "deploy": "gh-pages -d styleguide",
-    "test": "jest",
-    "test:update": "jest -u",
-    "test:watch": "jest --watchAll",
-    "test:coverage": "jest --coverage",
-    "eslint": "eslint --ext .js,.ts src/ ",
-    "eslint:fix": "eslint --ext .js,.ts src/ --fix "
+    "test": "lerna run test"
```

Comment:
unfortunately, the script is not working for me ;x

Reply:
do you think we should add something else here?
wouldn't it be worth to add `"docs": "lerna run styleguide"`?

Reply:
> unfortunately, the script is not working for me ;x

`npm run test` in root dir doesn't work?